integration-cli/docker_api_exec_resize_test.go

```diff
@@ -97,9 +98,5 @@ func (s *DockerSuite) TestExecResizeImmediatelyAfterExecStart(c *check.C) {
 	}
 
 	wg.Wait()
-	select {
-	case err := <-ch:
-		c.Fatal(err.Error())
-	default:
-	}
+	c.Assert(<-ch,checker.IsNil)
```

Comment:
This will block forever if no error occured.
If you want to remove the select statement you have to close the chan before this assertion and after the `wg.Wait`.